crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -207,41 +200,14 @@ impl HostFnBuilder {
 }
 
 type ContextMap = Lazy<Arc<Mutex<HashMap<u64, Arc<CallContext>>>>>;
-/// Map from an instance to it's context for a call.
+// Map from a context key to a call context. Call contexts are passed to host
+// fn calls for execution.
 static CONTEXT_MAP: ContextMap = Lazy::new(Default::default);
```

Comment:
the context map only serves for storing call contexts in memory now, not as keys for the instance cache any more